<launch>
  <arg name="robot_id" value="$(optenv ROBOT_ID unnamed_robot)"/>

  <group ns="$(arg robot_id)">
    <node pkg="sdpo_ratf_ros_localization"
        type="sdpo_ratf_ros_localization_node" name="sdpo_ratf_ros_localization"
        clear_params="true" output="$(optenv ROBOT_OUTPUT screen)">
      <rosparam command="load" subst_value="True"
          file="$(find sdpo_ratf_ros_localization)/config/sdpo_ratf_ros_localization.yaml"/>
    </node>
  </group>
</launch>
